src/grouping-and-mapping/clients/GroupsClient.ts

```diff
@@ -48,7 +48,7 @@ export class GroupsClient extends OperationsBase implements IGroupsClient {
   }
 
   public async updateGroup(accessToken: AccessToken, mappingId: string, groupId: string, group: GroupUpdate): Promise<Group> {
-    if (null == group.groupName && null == group.description && null == group.query) {
+    if (!group.groupName && !group.description && !group.query && !group.metadata) {
```

Comment:
`!"" === true`. Empty description is a valid update.

Reply:
Fixed